book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
 # The PEcAn XML {#pecanXML}
 
-The PEcAn system is configured using a xml file, often called settings.xml. The configuration file contains the following sections:
-1. [Info]
-2. [PEcAn Folders]
-3. [Database Access]
-4. [BETY Configuration](#bety-database-configuration)
-5. [Brown Dog](#brown-dog-configuration)
-6. [PFT Selection](#pft-selection)
-7. [Meta Analysis](#meta-analysis)
-8. [Ensemble Runs](#ensemble-runs)
-9. [Sensitivity Runs](#sensitivity-runs)
-10. [Model Setup](#model-setup)
-11. [Run Setup](#run-setup)
-12. [State Data Assimilation](#state-data-assimilation-tags)
-13. [Parameter Data Assimilation]
-14. [Benchmarking]
-
-## Info
-The information section, tagged `info`, contains information specific to your run. They are not critical settings, so any missing values will not break your run. 
-
-An example of this section looks like:
+The PEcAn system is configured using a XML file, often called `pecan.xml`.
+It contains the following major sections ("nodes"):
+
+- [Core configuration](#xml-core-config)
+	- [Top level structure](#xml-structure)
+	- [`info`](#xml-info) -- Run metadata
+	- [`outdir`](#xml-outdir) -- Output directory
+	- [`database`](#xml-database) -- Bety and other database settings
+	- [`pft`](#xml-pft) -- Plant functional type selection
+	- [`meta.analysis`](#xml-meta-analysis) -- Trait meta analysis
+	- [`model`](#xml-model) -- Model configuration
+	- [`run`](#xml-run) -- Run setup
+	- [`host`](#xml-host) -- Host information for remote execution
+- [Advanced features](#xml-advanced)
+	- [`ensemble`](#xml-ensemble) -- Ensemble runs
+	- [`sensitivity.analysis`](#xml-sensitivity-analysis) -- Sensitivity analysis
+	- [`parameter.data.assimilation`](#xml-parameter-data-assimilation) -- Parameter data assimilation
+	- (experimental) [`state.data.assimilation`](#xml-state-data-assimilation) -- State data assimilation
+	- (experimental) [`browndog`](#xml-browndog) -- Brown Dog configuration
+	- (experimental) [`benchmarking`](#xml-benchmarking) -- Benchmarking
+	
```

Comment:
Where's the documentation on MultiSettings runs and using sitegroups?

Reply:
Punting on this. Someone with more familiarity should write that documentation.